components/Relationships/RelationshipElement.tsx

```diff
@@ -2,35 +2,118 @@ import Link from 'next/link';
 import { RelationshipPerson } from 'types';
 
 interface Props {
-  title: string;
-  data: RelationshipPerson[];
+  type: string;
+  persons: RelationshipPerson[];
 }
 
-const RelationshipElement = ({ title, data }: Props): React.ReactElement => {
-  return data.length > 0 ? (
+const RelationshipElement = ({ type, persons }: Props): React.ReactElement => {
+  return (
     <div className="govuk-summary-list__row">
-      <dt className="govuk-summary-list__key">{title}</dt>
+      <dt className="govuk-summary-list__key">{renderTitle(type)}</dt>
       <dd className="govuk-summary-list__value">
         <ul className="govuk-list">
-          {data.map((person, i) => {
-            return (
-              <li className="lbh-link" key={`rel_${i}`}>
-                {person.id ? (
-                  <Link href={`/people/${person.id}`}>
-                    {`${person.firstName} ${person.lastName}`}
-                  </Link>
-                ) : (
-                  `${person.firstName} ${person.lastName}`
-                )}
-              </li>
-            );
-          })}
+          {persons
+            .sort(
+              (a, b) =>
+                b.lastName.localeCompare(a.lastName) ||
+                b.firstName.localeCompare(a.firstName)
+            )
+            .map((person, i) => {
+              return (
+                <li className="lbh-link" key={`rel_${i}`}>
+                  {person.id ? (
+                    <Link href={`/people/${person.id}`}>
+                      {`${person.firstName} ${person.lastName}`}
+                    </Link>
+                  ) : (
+                    `${person.firstName} ${person.lastName}`
+                  )}
+                </li>
+              );
+            })}
         </ul>
       </dd>
     </div>
-  ) : (
-    <></>
   );
 };
 
+function renderTitle(title: string) {
```

Comment:
Does this make more sense to be an object type rather than a massive `switch`? Something like:

```tsx
const mappings = {
  parents: 'Parents',
  parent: 'Parent',
  children: 'Children',
  // .. and so on ..
} as const;

const getTitleString = (relationshipType: keyof typeof mappings): string => {
  return mappings[relationshipType];
};
```

That magic `keyof typeof mappings` also means you get a nice type signature for your `getTitleString` method. That might be helpful, it might not, just a though 😄

Reply:
This sounds much better, thanks!
Will definitely implement this :)